src/policy/truc_policy.cpp

```diff
@@ -203,7 +203,7 @@ std::optional<std::pair<std::string, CTransactionRef>> SingleV3Checks(const CTra
 
     // Remaining checks only pertain to transactions with unconfirmed ancestors.
     if (mempool_ancestors.size() > 0) {
-        // If this transaction spends V3 parents, it cannot be too large.
+        // If this transaction spends TRUC parents, it cannot be too large.
         if (vsize > V3_CHILD_MAX_VSIZE) {
             return std::make_pair(strprintf("v3 child tx %s (wtxid=%s) is too big: %u > %u virtual bytes",
```

Comment:
nit: the v3 mentions in debug strings should be explicitly stated as version=3 for clarity.
here and other places
```suggestion
            return std::make_pair(strprintf("version=3 child tx %s (wtxid=%s) is too big: %u > %u virtual bytes",
```

Reply:
updated all